README.md

```diff
@@ -49,9 +49,9 @@ Copyright & Licensing
 
 DOMjudge is Copyright (c) 2004 - 2025 by the DOMjudge developers and
 all respective contributors. The current DOMjudge developers are Jaap
-Eldering, Nicky Gerritsen, Keith Johnson, Thijs Kinkhorst, Mart
-Pluijmaekers, Michael Vasseur and Tobias Werth; see the manual for
-a complete list of contributors.
+Eldering, Nicky Gerritsen, Kevin Jilissen, Keith Johnson, Thijs Kinkhorst,
+Mart Pluijmaekers, Michael Vasseur & Tobias Werth;
```

Comment:
not that I will block the merge of this PR, but I personally like `and` better than `&`